### PR TITLE
updated event highlights

### PR DESCRIPTION
**Pull request by khxia**
#135 

Will update the Hacksprint discord link and hoth application link once they are ready

### CODE REVIEW

**Comment by jodymlin** [comment]
Oh wow ahaha the HOTH 7 banner looks very out of place right now. Hmmm @khxia @GalenWong what do you think? I'm thinking of leaving it on for now because I want the HOTH blurb up, but do you think it looks bad?

File: src/data/events/highlights.js [hunk 1]
```diff
@@ -14,35 +13,32 @@ const hackFamDescription = `
...
 	},
 	{
-		name: 'Bruin Quest',
-		description: bruinQuestDescription,
-		imgFilePath: 'event/2020f-bquest-banner.png',
-		button: 'Website',
-		link: 'https://quest.uclaacm.com/'
+		name: 'Hack Off the Hill 8',
+		description: hothDescription,
+		imgFilePath: 'event/2020w-hoth-banner.png',
```
Comment (jodymlin) [inline src/data/events/highlights.js:40]: While we wait for the new banner, can we use this plain version of the banner instead? 
hoth-plain-banner

The current one has the date/location/etc. I think this one will be slightly better for the temporary banner.

**Review by jodymlin — CHANGES_REQUESTED** [review]
Looks mostly good to me 👍 Just a small request to change the temporary banner used for HOTH before we push.

File: src/data/events/highlights.js (hunk 1 above)
Comment (khxia) [reply src/data/events/highlights.js:40]: Will do so. Also, probably need to clean up all the old banners at some point

Comment (jodymlin) [reply src/data/events/highlights.js]: True. I'll make an issue to clean up our banner files.

**Review by jodymlin — APPROVED** [review]

**Pull request merged**